docs/install/iso-install.md

```diff
@@ -33,7 +33,7 @@ Note: This [video](https://youtu.be/X0VIGZ_lExQ) shows a quick overview of the I
 1. Configure the `Virtual IP` which you can use to access the cluster or join other nodes to the cluster <small>(Note: If your IP address is configured via DHCP, you will need to configure static MAC-to-IP address mapping on your DHCP server in order to have a persistent Virtual IP)</small>.
 1. Configure the `cluster token`. This token will be used for adding other nodes to the cluster.
 1. Configure the login password of the host. The default SSH user is `rancher`.
-1. (Optional) Configure the NTP Servers of the node. This defaults to `0.suse.pool.ntp.org`.
+1. (Recommended) Configure the NTP Servers of the node. This defaults to `0.suse.pool.ntp.org`.
```

Comment:
```suggestion
1. Recommended configuring the NTP server to make sure all nodes' times are synchronized. This defaults to `0.suse.pool.ntp.org`
```

Reply:
also need to update the same paragraph on the [index page](https://github.com/harvester/docs/blob/main/docs/index.md).

Reply:
Fixed it. Thank you.